pypeit/spectrographs/keck_hires.py

```diff
@@ -107,6 +107,15 @@ def default_pypeit_par(cls):
         par['calibrations']['slitedges']['overlap'] = True
         par['calibrations']['slitedges']['dlength_range'] = 0.25
 
+        # TODO: Order extrapolation is OFF by default.  Need to test how well adding orders plays with trying to account for overlap...
```

Comment:
remove?

Reply:
I'm going to leave this for a future PR from @debora-pe.  I think she will know how best to set these given her testing.